client/site/site.less

```diff
@@ -663,6 +663,33 @@ body.welcome {
       }
     }
   }
+
+  .donation-banner {
+    display: flex;
+    margin-top: -50px;
+    height: 50px;
+
+    a {
```

Comment:
The pure black color could be considered a subtle form of highlighting... if the main text above were off-black. Will do.